enterprise/frontend/src/embedding-sdk/components/public/StaticDashboard/StaticDashboard.tsx

```diff
@@ -0,0 +1,85 @@
+import { pick } from "underscore";
+
+import {
+  DEFAULT_EMBED_DISPLAY_OPTIONS,
+  useDashboardFullscreen,
+  useDashboardRefreshPeriod,
+  useEmbedTheme,
+  useRefreshDashboard,
+} from "metabase/dashboard/hooks";
+import type {
+  EmbedDisplayParams,
+  RefreshPeriod,
+} from "metabase/dashboard/types";
+import { isNotNull } from "metabase/lib/types";
+import { PublicDashboard } from "metabase/public/containers/PublicDashboard/PublicDashboard";
+import { Box } from "metabase/ui";
+import type { DashboardId, ParameterId } from "metabase-types/api";
+
+export const StaticDashboard = ({
+  dashboardId,
+  parameterValues,
+  refreshPeriod: initialRefreshPeriod = null,
+  bordered,
+  titled,
+  theme: userTheme,
+  font,
+  hideDownloadButton,
+  hideParameters,
+}: {
+  dashboardId: DashboardId;
+  parameterValues: Record<ParameterId, string | string[] | null | undefined>;
+  refreshPeriod?: RefreshPeriod;
+} & Partial<EmbedDisplayParams>) => {
+  const options: EmbedDisplayParams = {
+    ...DEFAULT_EMBED_DISPLAY_OPTIONS,
```

Comment:
nit: This var location made sense when it was only used inside the `dashboard/hooks`, but now it's being used outside that scope, it feels a bit out of place.

I won't suggest changing anything right now, but I can see if we keep referring to this constant outside the hook directory, we should start thinking about moving it out of there. So it's great to keep this in mind.

What do you think?

Reply:
Yeah, I agree. I'll keep it in the back of my mind while we continue on this journey